go-fuzz-build/main.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 var (
+	flagTag  = flag.String("tags", "", "a list of build tags to consider satisfied during the build.")
```

Comment:
s/a list/a space-separated list/ please
also no dot at the end, other flags don't use dots